api/version.proto

```diff
@@ -136,6 +136,10 @@ message VersionMatrix {
   map<string, Version> pgbouncer = 13;
   map<string, Version> pxc_operator = 14;
   map<string, Version> psmdb_operator = 15;
+  map<string, Version> operator_apiserver = 16;
+  map<string, Version> operator_event = 17;
+  map<string, Version> operator_rmdata = 18;
+  map<string, Version> operator_scheduler = 19;
```

Comment:
what about `deployer`?

Reply:
I don't think it's needed here. @hors what do you think?

Reply:
done